wazo_calld/plugins/calls/http.py

```diff
@@ -274,11 +276,25 @@ def put(self, call_id):
 
 class ConnectCallToUserResource(AuthResource):
     def __init__(self, calls_service):
-        self.calls_service = calls_service
+        self.calls_service: CallsService = calls_service
 
     @required_acl('calld.calls.{call_id}.user.{user_uuid}.update')
     def put(self, call_id, user_uuid):
-        new_call_id = self.calls_service.connect_user(call_id, user_uuid)
+        body = None
+        if request.data:
+            body = connect_call_request_body_schema.load(request.json)
+
+        args = {
+            'call_id': call_id,
+            'user_uuid': user_uuid,
+        }
+
+        if body:
+            # NOTE(clanglois): setting the 'timeout' to null
+            # allows disabling the timeout(infinite ringing)
```

Comment:
This important info should go in the documentation, not in a comment.

Reply:
why not both?
Agreed, I'll add it to the openapi doc.